### DISCUSSION TITLE
Error -2 connecting to redis-cache:6379. Name does not resolve.

### DISCUSSION THREAD

**Discussion by ipflowss** [netbox-community/netbox#9205] [General]
Hi there,
I am administering netbox via docker containers.  Its been running for a few weeks without me logging into it.  And now I see the following error.  I restarted all the conatiners and that did not clear the error.  What else can I check?

--- snip ---
<class 'redis.exceptions.ConnectionError'>

Error -2 connecting to redis-cache:6379. Name does not resolve.

Python version: 3.9.5
NetBox version: 3.1.8

--- snip ---

--- snip ---
❯ docker ps -a | grep netbox
18c9a16a21c7   netboxcommunity/netbox:v3.1-1.5.1   "/sbin/tini -- /opt/…"   11 days ago    Up 38 minutes   0.0.0.0:8000->8080/tcp, :::8000->8080/tcp       netbox-docker_netbox_1
2cc58583850c   netboxcommunity/netbox:v3.1-1.5.1   "/sbin/tini -- /opt/…"   11 days ago    Up 38 minutes                                                   netbox-docker_netbox-worker_1
a598831a63fd   netboxcommunity/netbox:v3.1-1.5.1   "/sbin/tini -- /opt/…"   11 days ago    Up 38 minutes                                                   netbox-docker_netbox-housekeeping_1
460d01e3d1f3   postgres:14-alpine                  "docker-entrypoint.s…"   11 days ago    Up 38 minutes   5432/tcp                                        netbox-docker_postgres_1
629d4fc53482   redis:6-alpine                      "docker-entrypoint.s…"   11 days ago    Up 38 minutes   6379/tcp
--- snip ---

Cheers

**Comment by candlerb** [comment]
Netbox is trying to connect to a hostname "redis-cache" (presumably another container?) and now your system cannot not resolve "redis-cache" to an IP address.

The problem is within your docker / kubernetes / whatever container environment.
